### PR TITLE
add file

### PR DESCRIPTION
**Pull request by alemata**

### CODE REVIEW

File: lib/test.exs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+defmodule PhoenixCourse.UserControllerTest do
```
Comment (credoci) [inline lib/test.exs]: Modules should have a @moduledoc tag.

File: lib/test.exs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      ~r/Oops, something went wrong! Please check the errors below./
+    assert user_count == users_before
+  end
+
+  defp last_user do
+    Repo.one(from u in User, order_by: [desc: u.id], limit: 1)
+  end
+
+  defp user_count do
+    Repo.one( from u in User, select: count(u.id) )
```
Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      ~r/Oops, something went wrong! Please check the errors below./
+    assert user_count == users_before
+  end
+
+  defp last_user do
+    Repo.one(from u in User, order_by: [desc: u.id], limit: 1)
+  end
+
+  defp user_count do
+    Repo.one(  from u in User, select: count(u.id) )
```
Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 1 above)
Comment (credoci) [inline lib/test.exs]: Modules should have a @moduledoc tag.

File: lib/test.exs (hunk 3 above)
Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 1 above)
Comment (credoci-dev) [inline lib/test.exs]: Modules should have a @moduledoc tag.

File: lib/test.exs (hunk 3 above)
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 1 above)
Comment (credoci-dev) [inline lib/test.exs]: Modules should have a @moduledoc tag.

File: lib/test.exs (hunk 3 above)
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 1 above)
Comment (credoci-dev) [inline lib/test.exs]: Modules should have a @moduledoc tag.

File: lib/test.exs (hunk 3 above)
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 1 above)
Comment (credoci-dev) [inline lib/test.exs]: Modules should have a @moduledoc tag.

File: lib/test.exs (hunk 3 above)
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 1 above)
Comment (credoci-dev) [inline lib/test.exs]: Modules should have a @moduledoc tag.

File: lib/test.exs (hunk 3 above)
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    assert html_response(conn, 200) =~
+      ~r/Oops, something went wrong! Please check the errors below./
+    assert user_count == users_before
+  end
+
+  defp last_user do
+    Repo.one(from u in User, order_by: [desc: u.id], limit: 1)
+  end
+
+  defp user_count do
```
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    Repo.one(  from u in User, select: count(u.id) )
```
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test2.exs [hunk 6]
```diff
@@ -0,0 +1,9 @@
+defmodule PhoenixCourse.UserControllerTest do
+  @moduledoc false
+  use PhoenixCourse.ConnCase
+  alias PhoenixCourse.User
+
+  defp user_count do
+    Repo.one( from u in User, select: count(u.id) )
```
Comment (credoci-dev) [inline lib/test2.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 5 above)
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test2.exs (hunk 6 above)
Comment (credoci-dev) [inline lib/test2.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test2.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 5 above)
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test2.exs (hunk 6 above)
Comment (credoci-dev) [inline lib/test2.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 5 above)
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test2.exs (hunk 6 above)
Comment (credoci-dev) [inline lib/test2.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 5 above)
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test2.exs (hunk 6 above)
Comment (credoci-dev) [inline lib/test2.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 5 above)
Comment (credoci-dev) [inline lib/test.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test2.exs (hunk 6 above)
Comment (credoci-dev) [inline lib/test2.exs]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 5 above)
Comment (credoci) [inline lib/test.exs:51]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test2.exs (hunk 6 above)
Comment (credoci) [inline lib/test2.exs:7]: There is no whitespace around parentheses/brackets most of the time, but here there is.

Comment (credoci) [inline lib/test2.exs:7]: There is no whitespace around parentheses/brackets most of the time, but here there is.

File: lib/test.exs (hunk 5 above)
Comment (credoci) [inline lib/test.exs:51]: There is no whitespace around parentheses/brackets most of the time, but here there is.

**Pull request closed without merging**